Lib/fontTools/unicodedata/__init__.py

```diff
@@ -45,6 +44,439 @@
     "ot_tag_to_script",
 ]
 
+# Derived from http://www.unicode.org/Public/UNIDATA/BidiMirroring.txt
```

Comment:
Please put this in it's own file, like `Scripts.py` and import it like that one. Thanks.